pipenv/project.py

```diff
@@ -199,6 +199,10 @@ def sources(self):
         else:
             return [{u'url': u'https://pypi.python.org/simple', u'verify_ssl': True}]
 
+    @property
+    def source_urls(self):
```

Comment:
I wish I could find one place to remove the duplication, but none of the choices seemed particularly good to me.